Frends.OAuth.ParseToken/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [1.0.1] - 2022-08-04
+### Added
```

Comment:
This should be `Changed`, since you are changing existing behavior.
Please also update the version in `.csproj`.